src/xcode/ENA/ENA/Source/Services/CoronaTestService/__tests__/PCRTest+Mock.swift

```diff
@@ -31,6 +31,7 @@ extension PCRTest {
 			submissionTAN: submissionTAN,
 			keysSubmitted: keysSubmitted,
 			journalEntryCreated: journalEntryCreated,
+			certificateSupportedByPointOfCare: true,
```

Comment:
should be added as a parameter so it can be configured by anyone using this mock. And usually we go with false as a default, unless there is a specific reason not to do so